Quorum/tests/test_price_feed_providers.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+
+import Quorum.tests.utils as utils
+
+from Quorum.utils.chain_enum import Chain
+from Quorum.apis.price_feeds import PriceFeedData, ChainLinkAPI, ChronicleAPI, CoinGeckoAPI
+
+import json
+from pathlib import Path
+import shutil
+
+from typing import Generator
+
+
+EXPECTED_DIR = utils.EXPECTED_DIR / 'test_price_feed_providers'
+
+
+@pytest.fixture
+def tmp_cache() -> Generator[Path, None, None]:
+    cache = Path(__file__).parent / 'tmp_cache'
+    cache.mkdir()
```

Comment:
This might be flaky, I would recommend add a validation this dir does not exist and delete if it is somehow.

Reply:
Done